docs/apis-tools/migration-manuals/migrate-to-camunda-api.md

```diff
@@ -667,3 +667,152 @@ The following conventions apply to all attributes:
 [setting variables]: /apis-tools/camunda-api-rest/specifications/create-element-instance-variables.api.mdx
 [general changes]: #general-endpoint-changes
 [multi-tenancy]: /self-managed/concepts/multi-tenancy.md
+
+### Flownode instances
+
+#### Search flownode instances
+
+- **[V1 endpoint](../operate-api/specifications/search-4.api.mdx)**: `POST /v1/flownode-instances/search`
+- **[V2 endpoint](../camunda-api-rest/specifications/search-element-instances.api.mdx)**: `POST /v2/element-instances/search`
+
+<Tabs groupId="search-flownode-instances" defaultValue="input-adjustments" queryString values={
+[
+{label: 'Input adjustments', value: 'input-adjustments'},
+{label: 'Output adjustments', value: 'output-adjustments'},
+]
+}>
+
+<TabItem value='input-adjustments'>
+
+- Request structure changes as outlined in [general changes][].
+  - `searchAfter` has been moved under `page`.
+  - `size` is now the `limit` in the `page` object.
+- Renamed attributes in the `filter` object
+  - `key` of type `int64` - Use `elementInstanceKey` of type `string` instead.
+  - `flowNodeId` - Use `elementId` instead.
+  - `flowNodeName` - Use `elementName` instead.
+  - `incident` - Use `hasIncident` instead.
+- Adjusted attributes in the `filter` object
+  - `processInstanceKey` - Type changed from `int64` to `string`.
+  - `processDefinitionKey` - Type changed from `int64` to `string`.
+  - `incidentKey` - Type changed from `int64` to `string`.
+- Removed attributes from the `filter` object
+  - `startDate` - Can no longer be used for filtering.
+  - `endDate` - Can no longer be used for filtering.
```

Comment:
💭 I have created https://github.com/camunda/camunda/issues/33281 for this. Otherwise, it's a regression in V2. We need to change this guide again when this is fixed. I'll leave a comment in the issue to fix the guide as well as a follow-up. We can leave this here for now as it's currently correct.